etl_tests/test_expiry.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from etl.expiry import get_expiry_date, get_expiry_text, validate
+
+
+def test_validate():
+    assert validate("2021-12-31") is True
```

Comment:
We can probably skip this method, since it's not a public method, just a helper function